set.mm

```diff
@@ -677950,6 +677950,84 @@ collection and union ( ~ mnuop3d ), from which closure under pairing
       URAABTUAABCDEFGHIJKUBUCUD $.
   $}
 
+  ${
+    $d U v $.  $d f g i w z $.  $d f g i v z $.  $d U f g i u w $.
+    $( Express the predicate on ` U ` in ~ ismnu in a shorter form while
+       avoiding complicated definitions.  (Contributed by Rohan Ridenour,
+       8-Oct-2024.) $)
+    ismnushort $p
+      |- ( A. z e. U A. f e. ~P U E. w e. U
+        ( ~P z C_ ( U i^i w ) /\ ( z i^i U. f ) C_ U. ( f i^i ~P ~P w ) )
+      <-> A. z e. U ( ~P z C_ U /\ A. f E. w e. U
+        ( ~P z C_ w /\ A. i e. z ( E. v e. U ( i e. v /\ v e. f )
+          -> E. u e. f ( i e. u /\ U. u C_ w ) ) ) ) ) $=
```

Comment:
Do we have
```
      |- ( A. f e. ~P U E. w e. U
        ( ~P z C_ ( U i^i w ) /\ ( z i^i U. f ) C_ U. ( f i^i ~P ~P w ) )
      <-> ( ~P z C_ U /\ A. f E. w e. U
        ( ~P z C_ w /\ A. i e. z ( E. v e. U ( i e. v /\ v e. f )
          -> E. u e. f ( i e. u /\ U. u C_ w ) ) ) ) ) $=
```
? If true, then it should probably be stated that way instead.

Reply:
Yes and done.